lib/veda_idmatrix/request.rb

```diff
@@ -84,15 +84,13 @@ def id_matrix_operation
 
     email_address = (self.entity[:email_address])
 
-    medicare_details = {}
+    medicare_details = {
+      :'card-number' => (self.entity[:medicare_card_number]),
+      :'reference-number' => (self.entity[:medicare_reference_number]),
+      :'card-colour' => (self.entity[:medicare_card_color]),
+      :'date-of-expiry' => (self.entity[:medicare_card_expiry])
+    }
     unless self.entity[:medicare_card_number].blank?
```

Comment:
this is not valid syntax, you cannot have an un-terminated conditional with no block.